.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -6,36 +6,20 @@ body:
     attributes:
       value: |
         Please provide the following information.
-  - type: input
-    id: Zarr-version
-    attributes:
-      label: Zarr version
-      description: Value of ``zarr.__version__``
-      placeholder: v2.10.2, v2.11.3, v2.12.0, etc.
-    validations:
-      required: true
-  - type: input
-    id: Numcodecs-version
-    attributes:
-      label: Numcodecs version
-      description: Value of ``numcodecs.__version__``
-      placeholder: v0.8.1, v0.9.0, v0.10.0, etc.
-    validations:
-      required: true
-  - type: input
-    id: Python-version
-    attributes:
-      label: Python Version
-      description: Version of Python interpreter
-      placeholder: 3.10, 3.11, 3.12 etc.
-    validations:
-      required: true
-  - type: input
-    id: OS
-    attributes:
-      label: Operating System
-      description: Operating System
-      placeholder: (Linux/Windows/Mac)
+  - type: textarea
+    id: version-info
+    attributes:
+      label: Versions
+      description: Output of ``zarr.print_debug_info()``
+      placeholder: |
```

Comment:
```suggestion
      description: Output of ``zarr.print_debug_info()``. This will automatically be rendered as code, no need for backticks.
      render: shell
      placeholder: |
```